src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
You can simplify the filtering logic by using a single filter method with a conditional expression inside, following the instruction given in the task description:

```js
const men = people.filter(
  (p) => p.sex === 'm' && (!century || Math.ceil(p.died / 100) === century)
);
```